src/main/kotlin/no/nav/familie/tilbake/config/KafkaConfig.kt

```diff
@@ -35,7 +36,9 @@ class KafkaConfig(@Value("\${KAFKA_BROKERS:localhost}") private val kafkaBrokers
     private fun producerConfigs() =
             mapOf(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG to kafkaBrokers,
                   ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG to StringSerializer::class.java,
-                  ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to StringSerializer::class.java) + securityConfig()
+                  ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to StringSerializer::class.java,
+                  ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION to 1, // den sikrer rekkefølge
```

Comment:
Vi har ingen garanti for at consumerene får hendlesene i rekkefølge, så dette vil i praksis ikke ha så mye effekt, og det er også en performance kost og bruke denne. Hvis vi heller gjør familie-historikk idempotent og bruker timestamp til å avjgøre rekkefølge tror jeg det er bedre.

Reply:
byttet denne propertien med idemportence property. Det bør gå fint med denne konfigurasjonen siden vi bare bruker en partition.